configs/config.yml

```diff
@@ -0,0 +1,39 @@
+listeners:
+  default:
+    bind: :25565
+    #proxyProtocol:
+      
+      # Receive proxy protocol
+      #
+      #receive: true
+
+      # TODO
+      #
+      #trustedProxies:
+      #  - "127.0.0.1"
+
+
+servers:
```

Comment:
Are you still planning to separate server configs into different files per server?

Reply:
Yes, but for now I think we should keep it as simple as possible or do you think this feature is very important and should be done now?

Reply:
I think separating this server configs into different files should be done before the release of this version. But for now having it here is fine i guess.

Reply:
I changed it to be like in the old days. Let me know what you think